src/lambda_streams/Numbers.java

```diff
@@ -32,60 +38,76 @@ public static void main(String[] args) {
 
     }
```

Comment:
There are no calls to the lambdas in your 'main' method, to confirm they return the expected results.